sys/linux/bpf.txt

```diff
@@ -535,8 +527,9 @@ bpf_map_info {
 } [align[8]]
 
 bpf_get_btf_info_arg {
-	btf	fd_btf
-	len	len[info, int32]
+	btf	fd_btf	(in)
+	len	len[info, int32]	(in)
+# TODO: What is "in" about info?
```

Comment:
I don't fully understand what this TODO is about. Either extend it or remove.

Note: that pointers are always "in" because we need to generate the actual address, even if the _pointee_ is out.

Reply:
The pointee needed some changes. This is now solved ([here](https://github.com/google/syzkaller/pull/2008/files#diff-4ecef29d4d59413838d3104bbd572fc5R530-R538)).